app/person-escort-record/index.js

```diff
@@ -26,7 +25,7 @@ router.use(newApp.mountpath, newApp.router)
 
 // Define shared middleware
 router.use(protectRoute('person_escort_record:view'))
-router.use(setRecord('personEscortRecord', personEscortRecordService.getById))
+router.use(setRecord('personEscortRecord', 'personEscortRecord'))
```

Comment:
This needed a bit of rethinking, happy for someone to point out a better approach.

Reply:
Gah. Because this method doesn't expose the request object...

I think this seems like a good approach on the surface. I suppose the only thing we're relying on is that both services (or any others we may create) always have a `getById` method...

Reply:
Yes, that is the downside, though if it doesn't, things will error out fairly quickly which prompt some resolution either to this middleware or to the service.

Reply:
If we can't rely on the method being called `getById` (though I think that is a worthwhile convention to enforce), something like

`setRecord('personEscortRecord', req.services.personEscortRecord, 'getById')`
or
`setRecord('personEscortRecord', 'personEscortRecord', 'getById')` ?

Reply:
we can definitely pass it - I reckon we can make that change if it becomes necessary though.